datalad/distribution/install.py

```diff
@@ -61,16 +61,27 @@
 
 @build_doc
 class Install(Interface):
-    """Install a dataset from a (remote) source.
+    """Install one or many datasets from remote URL(s) or local PATH source(s).
 
-    This command creates a local :term:`sibling` of an existing dataset from a
-    (remote) location identified via a URL or path. Optional recursion into
+    This command creates local :term:`sibling`(s) of existing dataset(s) from
```

Comment:
there is a docbuild failure:

```

Warning, treated as error:
/opt/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages/datalad/distribution/install.py:docstring of datalad.distribution.install.Install.__call__:1:term not in glossary: 'sibling`(s) of existing dataset(s) from (remote) locations specified as URL(s) or path(s). Optional recursion into potential subdatasets, and download of all referenced data is supported. The new dataset(s) can be optionally registered in an existing :term:`superdataset'
make: *** [Makefile:50: html] Error 2
make: Leaving directory '/home/runner/work/datalad/datalad/docs'
Error: Process completed with exit code 2.
```

I think the plural s needs to be escaped like this (untested, but often done in the handbook):
```suggestion
    This command creates local :term:`sibling`\(s) of existing dataset(s) from
```

Reply:
Thank you!